cmd/check_container.go

```diff
@@ -88,6 +89,21 @@ var checkContainerCmd = &cobra.Command{
 			cfg.Scratch = true
 		}
 
+		if submit {
```

Comment:
why another `if submit {` when we have that on line 157? This diff might not be showing the entire picture, or I'm missing something.

Reply:
This was done so this logic lives closer to the GetProject method call.